rustfmt.toml

```diff
@@ -7,6 +7,7 @@ merge_derives = false
 # tidy only checks files which are not ignored, each entry follows gitignore style
 ignore = [
     "build",
+    "/vendor/",
```

Comment:
double check: should there be a leading `/` here?

Reply:
Yea, it's a gitignore pattern:

* `foo` — any file or directory named foo anywhere in the repo.
* `foo/` — any directory named foo anywhere in the repo.
* `/foo` — a file or directory named foo at the root of the repo.
* `/foo/` — a directory named foo at the root of the repo.

The vendor directory is created in the root.

Reply:
Ah ok; so not the root of the filesystem :D